charts/heimdall/README.md

```diff
@@ -0,0 +1,96 @@
+# Heimdall Application Dashboard
+
+This is a helm chart for [heimdall](https://github.com/heimdall/heimdall) leveraging the [Linuxserver.io image](https://hub.docker.com/r/linuxserver/heimdall/)
+
+## TL;DR;
+
+```shell
+$ helm repo add billimek https://billimek.com/billimek-charts/
+$ helm install billimek/heimdall
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+helm install --name my-release billimek/heimdall
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
```

Comment:
With helm3 being the default now, we can probably drop the `--purge` flag for the `helm delete` command.